### PR TITLE
Feature  profile details

### PR DESCRIPTION
**Pull request by nickycadavillo**

### CODE REVIEW

**Comment by Ricecoder** [comment]
Beautiful! Good job with the icons! :+1:

**Comment by jmcreasman** [comment]
Looks awesome! GitHub should hire you to design their site!

File: src/index.html [hunk 1]
```diff
@@ -12,28 +12,35 @@
 
         <link rel="stylesheet" href="css/normalize.css">
         <link rel="stylesheet" href="css/main.css">
+        <link rel="stylesheet" href="octicons/octicons.css">
```
Comment (kellymurray) [inline src/index.html]: You can use the CDN here instead of pushing the font files, etc.

File: src/index.html [hunk 2]
```diff
@@ -12,28 +12,35 @@
...
-        <li id="followers">1.1k Followers</li>
-        <li id="starred">2 Starred</li>
-        <li id="following">6 Following</li>
+        <li id="organization"><span class="octicon octicon-organization"></span> GitHub</li>
+        <li id="location"><span class="octicon octicon-location"></span>  San Francisco</li>
+        <li id="email"><span class="octicon octicon-mail"></span> octocat@github.com</li>
+        <li id="website"><span class="octicon octicon-link"></span> http://www.github.com/blog</li>
+        <li id="joined"><span class="octicon octicon-clock"></span> Joined on Jan 25, 2011</li>
       </ul>
+      <table class="status">
```
Comment (kellymurray) [inline src/index.html]: How can you do this without using `<table>`?

**Pull request merged**